py/test/selenium/selenium_test_suite.py

```diff
@@ -29,6 +30,7 @@ def suite():
         unittest.makeSuite(test_default_server.TestDefaultServer),
         unittest.makeSuite(test_google.TestGoogle),
         unittest.makeSuite(test_i18n.TestI18n),
+        unittest.makeSuite(test_remote_connection.TestRemoteConnection),
```

Comment:
this test suite isn't really used...

The python tests are currently built / run with selenium's custom build tool (crazy-fun) and here's the main definition for python:

https://github.com/SeleniumHQ/selenium/blob/master/py/build.desc

Basically to get a test to run, you'll need to add the test case into https://github.com/SeleniumHQ/selenium/tree/master/py/test/selenium/webdriver/common

But those all assume you're running with a particular browser... which for this we don't really need and you did the correct thing by making a unit test... which we don't really have a structure for running...

@davehunt @AutomatedTester halp?